### PR TITLE
Group pages file restructure

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/pages/dashboard/subDashboards/native-desktop/ui.test.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-import dashboardWithCompleteSections from '../../../../fixtures/dashboardWithCompleteSections.json';
+import dashboardWithCompleteSections from '../../../../../fixtures/dashboardWithCompleteSections.json';
 import nativeDesktopFixture from './fixtureData.json';
-import { apiLocalhost, apiPath, clientLocalhost } from '../../../test-utils/config';
+import { apiLocalhost, apiPath, clientLocalhost } from '../../../../test-utils/config';
 
-const mocks = () => {
-  nock(apiLocalhost)
+const mocks = async () => {
+  await nock(apiLocalhost)
```
Comment (alibombaye) [inline app/pages/dashboard/subDashboards/native-desktop/ui.test.js]: we never seem to see the end of these 😂

File: app/pages/dashboard/ui.test.js [hunk 2]
```diff
@@ -292,7 +292,7 @@ test('should render the Error summary containing all the sections that failed th
   await t
     .expect(errorSummary.exists).notOk()
     .click(submitForModerationButton)
-
+  // below is failing - AssertionError: expected false to be truthy
```
Comment (alibombaye) [inline app/pages/dashboard/ui.test.js]: let's skip this test and put the comment at the top. We are not doing a submitForModeration for now.

Comment (kat1906) [reply app/pages/dashboard/ui.test.js]: actually it is fine now but forgot to remove the comment

Comment (alibombaye) [reply app/pages/dashboard/ui.test.js]: 😂

**Pull request merged**